packages/cyberstorm/src/components/componentStyles/Breadcrumb.module.css

```diff
@@ -0,0 +1,24 @@
+/* Common Breadcrumb styles */
+.breadcrumb {
+  display: flex;
+  flex-direction: row;
+  gap: var(--size-s);
+  align-items: center;
+  font-size: var(--size-m);
+  font-family: Inter, sans-serif;
+}
+
+.breadcrumb > a {
+  display: flex;
```

Comment:
Force-flexing the content of the <a> (which allows any ReactNode) seems kinda risky. It may work for us, but pretty much makes it pointless to type the `label` prop as ReactNode.

Reply:
Hmm, I'm not 100% sure if I understood the problem here correctly, because I can't explain my understanding or answer with a fix suggestion.

Let me know if the fix wasn't correct or if it might seem like I've missed the point.

I removed the force-flexing from the `.label`, since it doesn't need that. I also removed the forced `flex-direction: row` from `.root`.

Should the the `flex` stuff be defined in the stylings of `LinkLibrary`s `Link` function? For example in the `storybook-cyberstorm`s `LinkLibrary` file?

Reply:
I guess that's a poorly worded comment since I'm not 100% percent certain what I meant without seeing the whole old context, which is now gone. I think my point was the typing allowed the breadcrumb to accept any React node as a prop and show it. Which means one could pass e.g. a `a` that contains a `div` which contains a `h3`, `p`, and an icon, with their own CSS module defining the styles that should be used to render that component. But since you used `flex` there, it meant that component's styles would perhaps be flexed, even if whoever created that component didn't want them to. (Today I'm not 100% how this would actually work with the CSS module scoping.)

But the very point seems moot with the current version of the code. Feel free to mark this conversation resolved once you've read this.

Reply:
Ah alright, now I see what you meant. I'm not also that sure if it would have been a problem, cause how I've understood the flex inheritance it would have not been passed down. In any case the "flex forcing" there was not needed anyways, so it's good that it was removed.

Marking as resolved 👍